src/api/v2/v2AzureResourcesApi.ts

```diff
@@ -188,6 +188,23 @@ export interface ApplicationResource extends ResourceBase {
     readonly tags?: {
         [propertyName: string]: string;
     };
+
+    /**
+     * A copy of the raw resource.
+     */
+    readonly raw: {};
```

Comment:
What's the typing on the resources returned from Azure? `GenericItem` or something like that?

Reply:
It's `GenericResource`, but we also use this for a resource group, which we doesn't satisfy `GenericResource`.